src/tools/importPlanGoals.js

```diff
@@ -135,26 +145,53 @@ export default async function importGoals(fileKey, region) {
             const dbGrant = await Grant.findOne({ where: { ...fullGrant }, attributes: ['id', 'recipientId'] });
             if (!dbGrant) {
               // eslint-disable-next-line no-console
-              console.log(`Couldn't find grant: ${fullGrant.number}. Exiting...`);
+              logger.error(`Couldn't find grant: ${fullGrant.number}. Exiting...`);
```

Comment:
Do many of these occur? Does this indicate insufficiencies in the import of HSES? Are we missing a type of grant?